### ISSUE TITLE
Minard mainfont not taking properly

### ISSUE THREAD

**Issue by jimjam-slam** [qmd-lab/closeread#114]
Something I observed polishing up the YAML options is that the `mainfont` option appears to be escaped differently to how our font options are. It's causing the custom font for [the Minard example](https://preview.closeread.dev/gallery/examples/minards-map/) not to take properly:

![image](https://github.com/user-attachments/assets/3612c223-0e2b-49cd-be8c-571a919e6b33)

vs.

![image](https://github.com/user-attachments/assets/8e3741b9-f7dc-447c-9da8-86609b1a059b)

In the doc root, you can observe the variables:

```
--cr-narrative-font-family: et-book, Palatino, “Palatino Linotype”, “Palatino LT STD”, “Book Antiqua”, Georgia, serif;

[...]

--bs-body-font-family: et-book, Palatino, "" Palatino Linotype "", "" Palatino LT STD "", "" Book Antiqua "", Georgia, serif;
```

Not really sure why this is, or if I'm escaping improperly in the YAML 🤔

**Comment by andrewpbray** [comment]
Good catch. Something we should be able to figure out with some sleuthing! I'll go ahead and move this to post-contest (or at least post-announcement), since it's non-urgent.

**Comment by jimjam-slam** [comment]
I'm actually staring to suspect, based on the [HTML reference](https://quarto.org/docs/reference/formats/html.html#fonts), that the `mainfont` option is not designed to take a full comma-separated list of fallback fonts.

Following this up with the Quarto team!

**Comment by jimjam-slam** [comment]
Have filed as https://github.com/quarto-dev/quarto-cli/discussions/11268 (not an issue because I'm not sure whether it's intended behaviour from Quarto)

**Comment by jimjam-slam** [comment]
I think the short-term might just be to list `mainfont: et-book` alone, though!

**Comment by andrewpbray** [comment]
When I try this, I'm getting the navbar in `et-book`; it's overwriting `cosmo`, which makes sense. Should we just return to calling the font in the css file?

**Comment by jimjam-slam** [comment]
This is now fixed in the latest Quarto pre-release: it turns out that they do the quoting for you, and if you quote font names yourself, they were doubling them up. The solution was to simply leave the quotes out and let Quarto add them, but as of 1.6.34 this is fixed.

That said, as you say, specifying `mainfont` on the doc overwrites the font for the navigation chrome too, not just the article. So I agree: let's return to specifying the custom font in the CSS :)

**Comment by andrewpbray** [comment]
Ahhh, interesting. Well good to know!
